bin/meli-test-account.js

```diff
@@ -1,23 +1,12 @@
 #!/usr/bin/env node
 require('dotenv').config({path: require('path').resolve(process.cwd(), '.env.test')})
-require('../config/db').connect()
 
-const app = require('express')()
-const req = require('request-promise')
-const chromeLauncher = require('chrome-launcher')
-const {Spinner} = require('cli-spinner')
+const db = require('../config/db')
 
-const auth = require('../routes/auth')
-const Account = require('../model/account')
-const {refresh} = require('../config/meli-auth')
+const cliLoginFlow = require('./cli-meli-accounts/cli-login-flow')
```

Comment:
Module names are redundant, if you already are at cli folder you're already know that is all cli related, this could easily be `login.js` or `login-flow.js`. Leave name as simple as posible